File: beecrowd/1805/1805 - Natural Sum .cs
using System;

class Program
{
	static ulong Sum(ulong n)
	{
		ulong sum = (n * (n + 1)) / 2;
		return sum;	
	}
	
	static void Main(string[] args)
	{
		string[] input;
		ulong from, to, rSum;
		
		input = Console.ReadLine().Split(' ');
		from = ulong.Parse(input[0]);
		to = ulong.Parse(input[1]);
		
		rSum = Sum(to) - Sum(from - 1);
		Console.WriteLine(rSum);
	}
}

File: beecrowd/1805/1805 - Natural Sum .c
#include <stdio.h>

unsigned long long Sum(unsigned long long n)
{
	unsigned long long sum = (n * (n + 1)) / 2;

	return sum;
}

int main()
{
	unsigned long long from, to, rSum;
	scanf("%llu %llu", &from, &to);

	rSum = Sum(to) - Sum(from - 1);
	printf("%llu\n", rSum);

	return 0;
}
